.github/workflows/main.yml

```diff
@@ -148,8 +150,8 @@ jobs:
         run: ./script/build-and-sign.sh
       - name: Github release
         env:
-          RELEASES_API_KEY: ${{ secrets.RELEASES_API_KEY }}
-        run: ./script/github_releases.py "${STREAMLINK_DIST_DIR}"/*
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
there is no `secrets.GITHUB_TOKEN` in
https://github.com/streamlink/streamlink/settings/secrets/actions

Reply:
https://docs.github.com/en/actions/security-guides/automatic-token-authentication